iquip/apps/scheduler.py

```diff
@@ -204,10 +206,12 @@ def dropMimeData(self,  # pylint: disable=too-many-arguments
         mimedata: QMimeData,
         action: QAction,
         row: int,
-        column: int,
-        parentIndex: QModelIndex
+        column: int,  # pylint: disable=unused-argument
+        parentIndex: QModelIndex  # pylint: disable=unused-argument
```

Comment:
Since they are not used, I think the argument docstring for those be "Not used." unless you are going to use it later.